qt/widgets/instrumentview/src/ProjectionSurface.cpp

```diff
@@ -849,7 +849,11 @@ void ProjectionSurface::selectMultipleMasks(const QRect &rect) {
  */
 void ProjectionSurface::pickComponentAt(int x, int y) {
   size_t pickID = getPickID(x, y);
-  emit singleComponentPicked(pickID);
+  if (m_currentTab == 2) {
```

Comment:
Is there any way to have refer to the tabs by name? That would be safer in case the tab order is changed for whatever reason.